_includes/components/2-col.html

```diff
@@ -2,7 +2,21 @@
   <div class="one-account page-content__prose">
     {{ include.col1 | markdownify }}
   </div>
+  <div class="how-it-works page-content__prose">
+    <div class="banner-text">
+      {{ include.bannerText | markdownify }}
```

Comment:
small nit, not a huge deal, looks like none of our other include variables use camelCase, should we make this `bannertext` to match the others we have?

```bash
> git grep include\\. | perl -pe 's/^.*include\.([^ \}]+).*/\1/g' | sort | uniq
class
col1
col2
col3
dropdown
heading
id
mobile
size
subheading1
subheading2
subheading3
text
url
```